public/app/features/panel/components/PanelRenderer.tsx

```diff
@@ -1,18 +1,13 @@
 import React, { useState, useMemo, useEffect, useRef } from 'react';
-import { usePrevious } from 'react-use';
```

Comment:
Looks like `useRef` is not used here anymore.

Reply:
@Clarity-89 thanks!